packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -215,9 +215,9 @@ class ScrollPosition extends ViewportOffset {
   /// scroll notifications will be dispatched. No overscroll notifications can
   /// be generated by this method.
   ///
-  /// Immediately after the jump, a ballistic activity is started, in case the
-  /// value was out of range.
-  void jumpTo(double value) {
+  /// If settle is true then, immediately after the jump, a ballistic activity
+  /// is started, in case the value was out of range.
+  void jumpTo(double value, { bool settle: true }) {
```

Comment:
This was added to make the current ScrollNotification approach to synchronized scrolling work. The offset of the scroller that's following the leader is updated with jumpTo() each time the leader scrolls. In this case, because both scrollers have the same geometry, you want the follower to just mimic the leader's ballistic scroll.

This is all just a stopgap. When https://github.com/flutter/flutter/pull/8535 is sorted, I'll use that instead.